### PR TITLE
hitbtc: optimise fetching tradable pairs and tickers 

### PR DESCRIPTION
**Pull request by shazbert**
# PR Description

Fixes issue storing tradable pairs, fetching tickers and matching to available. 

Fixes # (issue)

## Type of change

Please delete options that are not relevant and add an `x` in `[]` as item is complete.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How has this been tested

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration and
also consider improving test coverage whilst working on a certain feature or package.

- [ ] go test ./... -race
- [ ] golangci-lint run
- [ ] Test X

## Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation and regenerated documentation via the documentation tool
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally and on Github Actions/AppVeyor with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by gloriousCode — APPROVED** [review]
tACK!
What a frustrating change: `quote := strings.Replace(symbols[x].ID, symbols[x].BaseCurrency, "", 1)`

File: exchanges/hitbtc/hitbtc_test.go [hunk 1]
```diff
@@ -161,33 +167,46 @@ func TestGetFeeByTypeOfflineTradeFee(t *testing.T) {
...
 		t.Fatal(err)
 	}
 	err = h.CurrencyPairs.StorePairs(asset.Spot, pairs, true)
 	if err != nil {
 		t.Fatal(err)
 	}
-	_, err = h.UpdateTicker(context.Background(),
-		currency.NewPair(currency.BTC, currency.USD),
-		asset.Spot)
+	_, err = h.UpdateTicker(context.Background(), currency.NewPair(currency.BTC, currency.USD), asset.Spot)
```
Comment (thrasher-) [inline exchanges/hitbtc/hitbtc_test.go:178]:
```suggestion
	_, err = h.UpdateTicker(context.Background(), pairs[0], asset.Spot)
```

File: exchanges/hitbtc/hitbtc_test.go [hunk 2]
```diff
@@ -161,33 +167,46 @@ func TestGetFeeByTypeOfflineTradeFee(t *testing.T) {
...
 	if err != nil {
 		t.Error(err)
 	}
-
-	_, err = h.FetchTicker(context.Background(),
-		currency.NewPair(currency.XRP, currency.USD), asset.Spot)
+	_, err = h.FetchTicker(context.Background(), currency.NewPair(currency.XRP, currency.USDT), asset.Spot)
```
Comment (thrasher-) [inline exchanges/hitbtc/hitbtc_test.go:182]:
```suggestion
	_, err = h.FetchTicker(context.Background(), pairs[1], asset.Spot)
```

File: exchanges/hitbtc/hitbtc_test.go (hunk 1 above)
Comment (thrasher-) [reply exchanges/hitbtc/hitbtc_test.go:178]: Might as well use `pairs` whilst we've got it 😄

File: exchanges/hitbtc/hitbtc_wrapper.go [hunk 3]
```diff
@@ -291,8 +291,9 @@ func (h *HitBTC) FetchTradablePairs(ctx context.Context, _ asset.Item) (currency
 
 	pairs := make([]currency.Pair, len(symbols))
 	for x := range symbols {
+		quote := strings.Replace(symbols[x].ID, symbols[x].BaseCurrency, "", 1)
```
Comment (thrasher-) [inline exchanges/hitbtc/hitbtc_wrapper.go:294]:
```
func TestFetchTradablePairs(t *testing.T) {
	if _, err := h.FetchTradablePairs(context.Background(), asset.Spot); err != nil {
		t.Fatal(err)
	}
}
```

**Review by thrasher- — CHANGES_REQUESTED** [review]
Just some minor changes, the pair display of "USD" vs "USDT" was a funny one!

**Review by thrasher- — APPROVED** [review]

**Pull request merged**